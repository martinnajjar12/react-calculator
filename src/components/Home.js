import { makeStyles, Typography } from '@material-ui/core';
import React from 'react';

const useStyles = makeStyles(theme => ({
  headerFontSize: {
    fontSize: '3rem',
    margin: '25px 0',
    [theme.breakpoints.down('xs')]: {
      fontSize: '2rem',
    },
  },
}));

const Home = () => {
  const classes = useStyles();
  return (
    <main>
      <Typography variant="h1" className={classes.headerFontSize}>Welcome to MathMagicians</Typography>
      <Typography>
        A mathemagician is a mathematician who is also a magician. The term
        mathemagic is believed to have been introduced by Royal Vale Heath
        with his 1933 book Mathemagic The name mathemagician was probably
        first applied to Martin Gardner, but has since been used to describe
        many mathematician/magicians, including Arthur T. Benjamin, Persi
        Diaconis, and Colm Mulcahy. Diaconis has suggested that the reason
        so many mathematicians are magicians is that inventing a magic trick and
        inventing a theorem are very similar activities. Mathemagician is a neologism,
        specifically a portmanteau, that combines mathematician and magician. A great
        number of self-working mentalism tricks rely on mathematical principles. Max
        Maven often utilizes this type of magic in his performance. The Mathemagician is
        name of a character in the 1961 children&apos;s book The Phantom Tollbooth.
        He is the ruler Digitopolis, the kingdom of mathematics.
      </Typography>
    </main>
  );
};

export default Home;
