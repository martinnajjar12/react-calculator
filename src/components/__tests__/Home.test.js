import renderer from 'react-test-renderer';
import Home from '../Home';

describe('Home component', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Home />);
    expect(tree).toMatchSnapshot();
  });
});
