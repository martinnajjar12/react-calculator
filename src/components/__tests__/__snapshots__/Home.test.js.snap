// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should match the snapshot 1`] = `
<main>
  <h1
    className="MuiTypography-root makeStyles-headerFontSize-1 MuiTypography-h1"
  >
    Welcome to MathMagicians
  </h1>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    A mathemagician is a mathematician who is also a magician. The term mathemagic is believed to have been introduced by Royal Vale Heath with his 1933 book Mathemagic The name mathemagician was probably first applied to Martin Gardner, but has since been used to describe many mathematician/magicians, including Arthur T. Benjamin, Persi Diaconis, and Colm Mulcahy. Diaconis has suggested that the reason so many mathematicians are magicians is that inventing a magic trick and inventing a theorem are very similar activities. Mathemagician is a neologism, specifically a portmanteau, that combines mathematician and magician. A great number of self-working mentalism tricks rely on mathematical principles. Max Maven often utilizes this type of magic in his performance. The Mathemagician is name of a character in the 1961 children's book The Phantom Tollbooth. He is the ruler Digitopolis, the kingdom of mathematics.
  </p>
</main>
`;
